kogito-dmn-quickstart/pom.xml

```
@@ -17,6 +17,8 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
+        <kogito.platform.group-id>org.kie.kogito</kogito.platform.group-id>
+        <quarkus.kogito.artifact-id>kogito-quarkus-bom</quarkus.kogito.artifact-id>
```

What about vvv to be in sync with quarkus platform properties ...
```suggestion
        <kogito.platform.artifact-id>kogito-quarkus-bom</kogito.platform.artifact-id>
```